src/Toolkit/Toolkit.WinUI/UI/Controls/PopupViewer/PopupViewer.Theme.xaml

```diff
@@ -1,69 +1,202 @@
-                                 Margin="0"
-                                 Padding="0"
-                                 Background="Transparent"
-                                 Text="{Binding FormattedValue, Mode=OneWay}"
-                                 Foreground="{ThemeResource AppBarItemForegroundThemeBrush}"
-                                 TextWrapping="Wrap" Grid.Row="1"/>
-                      </Grid>
-                    </DataTemplate>
-                  </ItemsControl.ItemTemplate>
-                </ItemsControl>
-                <TextBox IsReadOnly="True"
-                         BorderThickness="0"
-                         Margin="0"
-                         Padding="0"
-                         Background="Transparent"
-                         Text="{Binding CustomDescriptionHtml, Converter={StaticResource HtmlToPlainTextConverter}, Mode=OneWay}"
-                         Foreground="{ThemeResource AppBarItemForegroundThemeBrush}"
-                         Visibility="{Binding CustomDescriptionHtml, Converter={StaticResource VisibilityConverter}}"
-                         TextWrapping="Wrap"/>
-              </Grid>
-            </Grid>
-          </Border>
-        </ControlTemplate>
-      </Setter.Value>
-    </Setter>
-  </Style>
+                        <TextBlock Grid.Row="0" Grid.ColumnSpan="3" DataContext="{TemplateBinding Element}" Text="{Binding Title, Mode=OneWay}" Style="{StaticResource PopupViewerTitleStyle}" Visibility="{Binding Title, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+                        <TextBlock Grid.Row="1" Grid.ColumnSpan="3" DataContext="{TemplateBinding Element}" Text="{Binding Description, Mode=OneWay}" Style="{StaticResource PopupViewerCaptionStyle}" Visibility="{Binding Description, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+
+                        <StackPanel Grid.Row="2" Margin="0,0,0,20" DataContext="{TemplateBinding CurrentItem}" Grid.Column="1">
+                            <primitives:PopupMediaView PopupMedia="{Binding}" Margin="0,5" />
+                            <TextBlock Text="{Binding Title, Mode=OneWay}" Style="{StaticResource PopupViewerTitleStyle}" Visibility="{Binding Title, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+                            <TextBlock Text="{Binding Caption, Mode=OneWay}" Style="{StaticResource PopupViewerCaptionStyle}" Visibility="{Binding Caption, Converter={StaticResource PopupViewerVisibilityConverter}}" />
+                        </StackPanel>
+                        <Button Grid.Column="0" Grid.Row="2"  Content="" Background="Transparent" Height="40" BorderThickness="0" FontFamily="Segoe MDL2 Assets" VerticalAlignment="Center" x:Name="PreviousButton" />
+                        <Button Grid.Column="2" Grid.Row="2"  Content="" Background="Transparent" Height="40" BorderThickness="0" FontFamily="Segoe MDL2 Assets" VerticalAlignment="Center" x:Name="NextButton" />
```

Comment:
The icons on these buttons are basically invisible for me. Is something collapsing them?

![2024-08-02_012408 Toolkit SampleApp WinUI](https://github.com/user-attachments/assets/dd197d91-cd4e-436b-a085-a6a2b0955b24)